frontend/src/pages/new_question/JsonEditor.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState, useContext, useEffect } from 'react';
```

Comment:
I believe JSDoc can do this with the @file tag. I think this would only be useful in utility files, as component files are fairly straight forward (they contain one component). Regardless, the goal is to have documentation for every function.

Reply:
For a file with hundreds of lines and typical 2-space tabs, I don't want to have to scroll through and make sure that nothing is being done except the one component. Unless this is a very strong convention and the naming of the file makes it clear that it exports a component and nothing else.

Reply:
Opened an issue for this: https://github.com/NCATS-Gamma/robokop-rewrite/issues/178